kong/tools/responses.lua

```diff
@@ -96,7 +102,7 @@ local function send_response(status_code)
   -- @param content (Optional) The content to send as a response.
   -- @return ngx.exit (Exit current context)
   return function(content, headers)
-    if status_code >= _M.status_codes.HTTP_INTERNAL_SERVER_ERROR then
+    if status_code >= _M.status_codes.HTTP_INTERNAL_SERVER_ERROR and status_code ~= _M.status_codes.HTTP_SERVICE_UNAVAILABLE then
```

Comment:
Additionally, I wonder if this behavior should actually be changed to *only* log when we send 500, instead of 5xx.